test/lib/jdk/test/lib/Platform.java

```diff
@@ -189,6 +189,19 @@ public static String getVMVersion() {
         return vmVersion;
     }
 
+    public static boolean isMusl() {
```

Comment:
I think will need test/lib/TestMutuallyExclusivePlatformPredicates.java to be updated too.

Reply:
Hi Alan, thanks for the advice;  do you think we can put it into the IGNORED group ?
https://github.com/openjdk/jdk/blob/master/test/lib-test/jdk/test/lib/TestMutuallyExclusivePlatformPredicates.java#L53

Reply:
Its now on the ProblemList and Created Issue:
 [JDK-8286191](https://bugs.openjdk.java.net/browse/JDK-8286191) Test library test failure in TestMutuallyExclusivePlatformPredicates.java